custom/icds/serializers.py

```diff
@@ -19,7 +19,10 @@ class Meta(object):
 
     def to_representation(self, instance):
         ret = super(HostedCCZSerializer, self).to_representation(instance)
-        ret['app_name'] = self.context['app_names'][ret['app_id']]
+        if instance.build_doc:
+            ret['app_name'] = instance.build_doc['name']
+            custom_properties = instance.build_doc['profile'].get('custom_properties', {})
+            ret['app_version_tag'] = custom_properties.get('cc-app-version-tag')
```

Comment:
any reason to not chain that? readibility?
```
custom_properties = (instance.build_doc['profile'].
                                       get('custom_properties', {}).
                                       get('cc-app-version-tag'))
```
with correct indentation(github wont let me)

Reply:
Yeah, it was a line length issue. Going to leave this as-is; I don't really like non-4-space indents. (I use them all the time, but begrudgingly :-p )